### DISCUSSION TITLE
Add thirteen CioherOS branch?

### DISCUSSION THREAD

**Discussion by gitretard** [ROM-builders/temporary#13687] [General]
Source is buildable ig as it runs on some device they showed

**Comment by Apon77** [comment]
Maybe cipher?

**Reply by gitretard** [reply]
I mean I want to build the Thirteen branch But I am PR user and it's unsupported I've been late to the A13 party for my device

**Reply by gitretard** [reply]
*CipherOS

**Comment by Apon77** [comment]
do pr
